### PR TITLE
feature/link-to-admin-from-dashboard

### PR DESCRIPTION
**Pull request by dabeeeenster**
This is a QOL improvement when dealing with accounts etc.

### CODE REVIEW

File: api/sales_dashboard/templates/sales_dashboard/organisation.html [hunk 1]
```diff
@@ -75,8 +75,8 @@ <h2>Subscription</h2>
                         <tr>
                             <td>{{ organisation.subscription.subscription_date }}</td>
                             <td>{{ organisation.subscription.cancellation_date }}</td>
-                            <td>{{ organisation.subscription.subscription_id }}</td>
-                            <td>{{ organisation.subscription.customer_id }}</td>
+                            <td><a target="_blank" href="https://flagsmith.chargebee.com/customers?view_code=all&Customers.search={{ organisation.subscription.customer_id }}">{{ organisation.subscription.customer_id }}</a></td>
```
Comment (matthewelwell) [inline api/sales_dashboard/templates/sales_dashboard/organisation.html]: There's a chance these values could be null / invalid (in the case that an organisation is subscribed but pays via Xero for example). In which case you'll end up with an invalid link. Probably not a massive issue but pretty easy to clean up: 

```html
<td>{% if organisation.subscription.customer_id %}<a target="_blank" href="https://flagsmith.chargebee.com/customers?view_code=all&Customers.search={{ organisation.subscription.customer_id }}">{{ organisation.subscription.customer_id }}</a>{% endif %}</td>
<td>{% if organisation.subscription.subscription_id %}<a target="_blank" href="https://flagsmith.chargebee.com/subscriptions?view_code=all&Subscriptions.search={{ organisation.subscription.subscription_id }}">{{ organisation.subscription.subscription_id }}</a>{% endif %}</td>
```

**Review by matthewelwell — CHANGES_REQUESTED** [review]

File: api/sales_dashboard/templates/sales_dashboard/organisation.html (hunk 1 above)
Comment (dabeeeenster) [reply api/sales_dashboard/templates/sales_dashboard/organisation.html]: Ah yes - updated

Comment (matthewelwell) [reply api/sales_dashboard/templates/sales_dashboard/organisation.html]: Sorry, just thinking about it more, probably the better option is: 

```html
{% if organisation.subscription.payment_method == "CHARGEBEE" %}...{% endif %}
```

I imagine some of the Xero subscriptions still have dummy data in these fields.

Comment (dabeeeenster) [reply api/sales_dashboard/templates/sales_dashboard/organisation.html]: OK tweaked

**Review by matthewelwell — APPROVED** [review]

**Pull request merged**